app/src/main/java/org/wikipedia/readinglist/ReadingListItemView.kt

```diff
@@ -64,6 +72,12 @@ class ReadingListItemView : ConstraintLayout {
                         menu.menu.findItem(R.id.menu_reading_list_rename).isVisible = false
                         menu.menu.findItem(R.id.menu_reading_list_delete).isVisible = false
                     }
+                    actionMode = callback?.onLongClick()
```

Comment:
Would it be possible to set `actionMode` when creating the view instead of setting the value through the callback?

Reply:
We need to check at run time, at the time when the user long presses, to see if the actionmode is Search, and decide what to do.. so if i create the view with the actionmode at the time of creation, it will not work. However, with the design changes, I might not need to hide it during search. Will confirm this with Robin and remove this logic altogether. Thank you for flagging.